linera-service/tests/end_to_end_tests.rs

```diff
@@ -685,7 +685,7 @@ async fn test_end_to_end_social_user_pub_sub() {
         .make_application(&chain2, &application_id)
         .await
         .into();
-    let subscribe = format!("mutation {{ subscribe(chainId: \"{chain1}\") }}");
+    let subscribe = format!("mutation {{ requestSubscribe(field0: \"{chain1}\") }}");
```

Comment:
Is this `field_0` ? (same below)

Reply:
I'd rather change the app's operations to be tuple variants, so this is still called `chainId`. And maybe still `subscribe`.

Reply:
Ah, yes, of course! :facepalm: